### ISSUE TITLE
Error with version 1.0.0-beta.130

### ISSUE THREAD

**Issue by peruzzof** [Redocly/redocly-cli#1159]
**Describe the bug**
Running the command "redocly bundle redoc/openapi.yaml --output filename.json" generate an error "Cannot find module 'react'"

**To Reproduce**
1. With any openapi.yaml run the command to generate a bundle
2. You will see the error
**Expected behavior**

The file is generated

**Logs**

+ redocly bundle <redacted>.yaml --output <redacted>.json
internal/modules/cjs/loader.js:934
  throw err;
  ^

Error: Cannot find module 'react'
Require stack:
- C:\npm\prefix\node_modules\@redocly\cli\node_modules\redoc\bundles\redoc.lib.js
- C:\npm\prefix\node_modules\@redocly\cli\lib\commands\build-docs\index.js
- C:\npm\prefix\node_modules\@redocly\cli\lib\index.js
- C:\npm\prefix\node_modules\@redocly\cli\bin\cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:931:15)
    at Function.Module._load (internal/modules/cjs/loader.js:774:27)
    at Module.require (internal/modules/cjs/loader.js:1003:19)
    at require (internal/modules/cjs/helpers.js:107:18)
    at C:\npm\prefix\node_modules\@redocly\cli\node_modules\redoc\bundles\redoc.lib.js:2:96534
    at C:\npm\prefix\node_modules\@redocly\cli\node_modules\redoc\bundles\redoc.lib.js:1807:3444
    at C:\npm\prefix\node_modules\@redocly\cli\node_modules\redoc\bundles\redoc.lib.js:1807:3449
    at C:\npm\prefix\node_modules\@redocly\cli\node_modules\redoc\bundles\redoc.lib.js:2:81
    at Object.<anonymous> (C:\npm\prefix\node_modules\@redocly\cli\node_modules\redoc\bundles\redoc.lib.js:2:191)
    at Module._compile (internal/modules/cjs/loader.js:1114:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\npm\\prefix\\node_modules\\@redocly\\cli\\node_modules\\redoc\\bundles\\redoc.lib.js',
    'C:\\npm\\prefix\\node_modules\\@redocly\\cli\\lib\\commands\\build-docs\\index.js',
    'C:\\npm\\prefix\\node_modules\\@redocly\\cli\\lib\\index.js',
    'C:\\npm\\prefix\\node_modules\\@redocly\\cli\\bin\\cli.js'
  ]
}

**OpenAPI definition**

**Redocly Version(s)**

1.0.0-beta.130

**`Node.js` Version(s)**

**Additional context**
I am running the same command in multiple pipelines and all of them starting to fail today, with the newest version

**Comment by tatomyr** [comment]
Hi @peruzzof,

Could you provide extra details like NodeJS and OS versions, project dependencies, &c.? Are you using a globally installed Redocly CLI or as a project dependency?

**Comment by peruzzof** [comment]
I am using in an Azure DevOps pipeline, running in the "ubuntu-latest" image. The NodeJS is installed with:

- task: NodeTool@0
    displayName: 'Install Node.js'
    inputs:
      versionSpec: '14.x'

And the command to install redocly is "npm i -g @redocly/cli@latest"

Let me know if you need any more detail about the machine.

**Comment by tatomyr** [comment]
Thanks, I was able to reproduce the bug in **Docker/NodeJS@14/npm@6.14.** 
Could you check if using **npm@7+** solves the issue for you?

**Comment by peruzzof** [comment]
@tatomyr , I cannot specify the npm version but I was able to upgrade my Node version from 14.x to 18.x and the install is working.
Thanks

**Comment by TitaneBoy** [comment]
Having exactly the same issue.

OS: Windows 10
Node: v18.16.1
typescript: v5.1.6
npm: v9.5.1

redocly installed as a project dependency (i.e locally) with yarn v1.22.19

**Comment by tatomyr** [comment]
@TitaneBoy thanks for reporting. Meanwhile you can try installing the peer dependencies by yourself as a workaround:
`yarn add mobx@^6.0.4 react@^17.0.0 react-dom@^17.0.0 styled-components@^5.1.1`.

**Comment by ArthurFlageul** [comment]
I have the same issue, and the suggested workaround didn't work. I ended up pinning redocly to the previous version to fix it.

**Comment by ArthurFlageul** [comment]
It's a docker image based on golang:1.20.4 with:

- yarn v1.22.19
- Node.js v18.16.0

Failed with:

```
success Installed "@redocly/cli@1.0.0-beta.130" with binaries:
--
  | - openapi
  | - redocly
  | Done in 14.12s.
  | node:internal/modules/cjs/loader:1078
  | throw err;
  | ^
  |  
  | Error: Cannot find module 'react'
  | Require stack:
  | - /usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js
  | - /usr/local/share/.config/yarn/global/node_modules/@redocly/cli/lib/commands/build-docs/index.js
  | - /usr/local/share/.config/yarn/global/node_modules/@redocly/cli/lib/index.js
  | - /usr/local/share/.config/yarn/global/node_modules/@redocly/cli/bin/cli.js
  | at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
  | at Module._load (node:internal/modules/cjs/loader:920:27)
  | at Module.require (node:internal/modules/cjs/loader:1141:19)
  | at require (node:internal/modules/cjs/helpers:110:18)
  | at /usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:2:96534
  | at /usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:1807:3444
  | at /usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:1807:3449
  | at /usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:2:81
  | at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:2:191)
  | at Module._compile (node:internal/modules/cjs/loader:1254:14) {
  | code: 'MODULE_NOT_FOUND',
  | requireStack: [
  | '/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js',
  | '/usr/local/share/.config/yarn/global/node_modules/@redocly/cli/lib/commands/build-docs/index.js',
  | '/usr/local/share/.config/yarn/global/node_modules/@redocly/cli/lib/index.js',
  | '/usr/local/share/.config/yarn/global/node_modules/@redocly/cli/bin/cli.js'
  | ]
  | }
  |  
```

I'll update this message if you need more details.

**Comment by IgorKarpiuk** [comment]
Hi @peruzzof please check if the problem is solved in version `1.0.0-beta.131`
